### ISSUE TITLE
No click tracking in banner ads with MPBannerCustomEventAdapter

### ISSUE THREAD

**Issue by mobile-ads-github** [mopub/mopub-ios-sdk#316]
Hello

I'm using custom banner adapter for MoPub mediation with ```enableAutomaticImpressionAndClickTracking=false```. I've noticed that after release mopub-ios-sdk 5.12.0 clicks stop tracking. And I've noticed this change occurred  in ```MPBannerCustomEventAdapter``` in release 5.12.0
https://github.com/mopub/mopub-ios-sdk/commit/01105fa081855b1b71316791d03c927be7ebc92b
<img width="981" alt="Screen Shot 2020-04-10 at 4 53 40 PM" src="https://user-images.githubusercontent.com/31896236/79121310-a92f2c80-7d9d-11ea-8a56-79358d952575.png">

Looks like clicks now only get tracked with enableAutomaticImpressionAndClickTracking=true.
Can you please help me and provide correct way to report click to MPBannerCustomEventAdapter?

#### MoPub SDK Version:
5.12.0

#### Evidence:
Logs with MoPub 5.12.0
```
2020-04-13 15:25:07.200625+0300 AdsSample[6146:129050] 
	[MoPub][MoPubAdapterLogger][block] Adapter ad from BannerCustomEvent received tap event
2020-04-13 15:25:07.200872+0300 AdsSample[6146:129050] 
	[MoPub][MoPubAdapterLogger][block] Adapter ad from BannerCustomEvent will present modal
2020-04-13 15:25:07.201051+0300 AdsSample[6146:129050] 
	[MoPub][MPBannerAdManager][ad-unit-id] Ad tapped
2020-04-13 15:25:07.201224+0300 AdsSample[6146:129050] 
	[MoPub][MPBannerAdManager][ad-unit-id] Ad will present modal
```

Logs with MoPub 5.11.0
```
2020-04-13 15:40:05.117213+0300 AdsSample[7805:145136] 
	[MoPub][MPAnalyticsTracker] Tracking click: https://ads.mopub.com/m/aclk?appid=..........
2020-04-13 15:40:05.120041+0300 AdsSample[7805:145137] 
	[MoPub][MoPubAdapterLogger][block] Adapter ad from BannerCustomEvent received tap event
2020-04-13 15:40:05.120233+0300 AdsSample[7805:145137] 
	[MoPub][MoPubAdapterLogger][block] Adapter ad from BannerCustomEvent will present modal
2020-04-13 15:40:05.120508+0300 AdsSample[7805:145137] 
	[MoPub][MPBannerAdManager][ad-unit-id] Ad tapped
2020-04-13 15:40:05.120705+0300 AdsSample[7805:145137] 
	[MoPub][MPBannerAdManager][ad-unit-id] Ad will present modal
```

**Comment by dravyan** [comment]
@mobile-ads-github , thank you for reporting. We will take a look and get back to you.

Thanks,
MoPub Team

**Comment by chauduyphanvu** [comment]
@mobile-ads-github, this issue has now been addressed in [v5.12.1](https://github.com/mopub/mopub-ios-sdk/commit/a87e944187e8ee5ddbd236222345af96a0cd1e7a) of the iOS SDK. Upgrade to this version and let us know if you continue seeing the issue!

Vu Chau
MoPub Team
